src/zoid/checkout/component.jsx

```diff
@@ -232,9 +232,15 @@ export function getCheckoutComponent() : CheckoutComponent {
                 }
             },
         
-            dimensions: isDevice()
-                ? { width:  '100%', height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` }
-                : { width:  `${ DEFAULT_POPUP_SIZE.WIDTH }px`, height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` }
+            dimensions: ({ props }) => {
+                if (props.dimensions) {
```

Comment:
Could we type check here to ensure `props.dimensions.width` never errors
```suggestion
                if (typeof props.dimensions === 'object') {
```